### ISSUE TITLE
[14.0 SaaS] URGENT :  Regression on attachment fix d3a6a20788c78ddaaff0919ea627e7b10c7ab81c

### ISSUE THREAD

**Issue by JayVora-SerpentCS** [odoo/odoo#117443]
Impacted versions: 14.0 SaaS

Steps to reproduce:
Non admin user goes to make an update on newletter builder

Current behavior:
Shows error as not allowd to move ahead.

Expected behavior:
Should allow.

Video/Screenshot link (optional):
![image](https://user-images.githubusercontent.com/5909386/229447330-d3dcc768-47b2-4e77-bc35-c5a6cf7eb873.png)

Its a regression: https://github.com/odoo/odoo/commit/d3a6a20788c78ddaaff0919ea627e7b10c7ab81c

SAAS users cannot move ahead.

**Comment by JayVora-SerpentCS** [comment]
@vava-odoo It has been 3 days and we are stuck on saas!
cc @odony help please.

**Comment by vava-odoo** [comment]
Hi @JayVora-SerpentCS 

Did you create a ticket in the [support page](https://www.odoo.com/help) as this is the official way to report bugs?
If not, I can create one for you.

Also, are you sure about the commit you mention? Because this one was merged 3 months ago.

Regards,

**Comment by JayVora-SerpentCS** [comment]
We haven't touched database, its default odoo newsletter data of default images on templates.
Suggest a way to handle this. you mean to delete them all and upload?

I see the code and its a clear blocker nevertheless.
@vava-odoo pls tell me the support ticket you create, we will add up there.
Plus, we are on v14 saas where this seems broken.

Let me know.
Thanks for the support.

**Comment by vava-odoo** [comment]
Hi @JayVora-SerpentCS 

> @vava-odoo pls tell me the support ticket you create, we will add up there.

Well, I didn't, since you did not reply to my message one month ago. 
Feel free to open one if the suggested solution does not fit.
Regards.

**Comment by JayVora-SerpentCS** [comment]
Yes dear, it does not work. Please direct us, we cannot delete attachemnts at all. Its a production db.
Thanks,

**Comment by vava-odoo** [comment]
There is a ticket #3305716 for your case. Could you provide the name of the database? Thanks.
